File: Java/Basics To Java/src/D/D3.java
package D;

import java.util.Scanner;

public class D3 {
public static void mainx(String[] args) {
	Scanner sc = new Scanner(System.in);
	
    String vypis = "";
    int cisla, rozdil=0;

    System.out.println("zadej první èlen posloupnosti");
    int prvni = sc.nextInt();
    System.out.println("zadej druhý èlen posloupnosti");
    int druhy = sc.nextInt();
    System.out.println("zadej poslední èlen posloupnosti");
    int posledni = sc.nextInt();
    cisla = prvni;

    cisla = cisla + rozdil;
    vypis = vypis + cisla + " ";
    do
    {
        rozdil = druhy - prvni;
        cisla = cisla + rozdil;
        if (cisla <= posledni)
            vypis = vypis + cisla + " ";
    }
    while (cisla < posledni);

    System.out.println();
    System.out.println("lineární posloupnost: " + vypis);
}
}

File: C#/Basic logic/D/D3.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace D
{
    internal class D3
    {
        static void Mainx(string[] args)
        {
            string vypis = "";
            int cisla, rozdil=0;

            Console.WriteLine("zadej první člen posloupnosti");
            int prvni = int.Parse(Console.ReadLine());
            Console.WriteLine("zadej druhý člen posloupnosti");
            int druhy = int.Parse(Console.ReadLine());
            Console.WriteLine("zadej poslední člen posloupnosti");
            int posledni = int.Parse(Console.ReadLine());
            cisla = prvni;

            cisla = cisla + rozdil;
            vypis = vypis + cisla + " ";
            do
            {
                rozdil = druhy - prvni;
                cisla = cisla + rozdil;
                if (cisla <= posledni)
                    vypis = vypis + cisla + " ";
            }

            while (cisla < posledni);

            Console.WriteLine();
            Console.WriteLine("lineární posloupnost: " + vypis);

            //cisla = cisla + rozdil;
            //vypis = vypis + cisla + " ";
            //while (cisla < posledni)
            //{

            //    rozdil = druhy - prvni;
            //    cisla = cisla + rozdil;
            //    if (cisla <= posledni)
            //        vypis = vypis + cisla + " ";
            //}



            //Console.WriteLine();
            //Console.WriteLine("lineární posloupnost: " + vypis);
        }
    }
}
